WIP-CHANGELOG.md

```diff
@@ -1,19 +1,14 @@
 ### Release notes
 
-- Removed unused config `global.environmentName` and added `global.clusterName` to migrate there's [this script](migration/v0.8.x-v0.9.x/migrate-config.sh)
-- To udate the password for `user-alertmanager` you'll have to re-install the chart
-  `./bin/ck8s ops helmfile wc -l app=user-alertmanager destroy && ./bin/ck8s ops helmfile wc -l app=user-alertmanager apply`
+- Removed unused config `global.environmentName` and added `global.clusterName` to migrate there's [this script](migration/v0.8.x-v0.9.x/migrate-config.sh).
```

Comment:
Since this should already be in the changelog when this gets released, should we really need to have these changes in the wip-changelog?